src/ripple/overlay/impl/Tuning.h

```diff
@@ -54,16 +54,16 @@ enum
     checkSeconds        =   10,
 
     /** How often we latency/sendq probe connections */
-    timerSeconds        =    3,
+    timerSeconds        =    4,
 
     /** How many timer intervals a sendq has to stay large before we disconnect */
-    sendqIntervals      =    3,
+    sendqIntervals      =    4,
```

Comment:
These were picked by testing. Automatic tuning would be great, but it's way out of scope.